<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="App_control">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="connect"/>
    <widget class="QWidget" name="Ads">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>811</width>
        <height>631</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="PendingAds">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Pending Ads</string>
       </attribute>
       <widget class="QTableWidget" name="Pending_table">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>781</width>
          <height>521</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Traditional Arabic&quot;;
color:rgb(85, 0, 0);
background-color:rgb(170, 255, 255);</string>
        </property>
        <property name="textElideMode">
         <enum>Qt::TextElideMode::ElideMiddle</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="Reject_button_2">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>550</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Images/refresh.png</normaloff>:/Images/refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="Reject_button">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>550</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Images/delete.png</normaloff>:/Images/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="Approve_button">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>550</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images.qrc">
          <normaloff>:/Images/check.png</normaloff>:/Images/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ApprovedAds">
       <attribute name="title">
        <string>Other Ads</string>
       </attribute>
       <widget class="QTableWidget" name="AllAds_Table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>781</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Discount">
       <attribute name="title">
        <string>Discount</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>391</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF;
 border-radius: 10px;
 border: 1px solid #E0E0E0;</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>70</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>70</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255,0,0)</string>
        </property>
        <property name="text">
         <string>Coupon Code : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255,0,0)</string>
        </property>
        <property name="text">
         <string>Expiration Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>60</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255,0,0)</string>
        </property>
        <property name="text">
         <string>Percent:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="table_discounts">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>701</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Users"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
